README.md

```diff
@@ -33,6 +33,10 @@ The container expects the following environment variables:
 
 `PRIVATE_KEY` - Private key content
 
+`PASSWORD`  - Password in case you do not use a private key (not recommended)
```

Comment:
Here should be written as well, that `PASSWORD` overrides `PRIVATE_KEY`

Reply:
changed the text in d046daa